common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala

```diff
@@ -68,5 +68,19 @@ object DisplayNoteType {
         DisplayNoteType("location-of-original", "Location of original")
       case BindingInformation(_) =>
         DisplayNoteType("binding-detail", "Binding detail")
+      case BiographicalNote(_) =>
```

Comment:
What's the difference between this and `BibliographicalInformation `?

Reply:
bibliographical vs biographical

Reply:
Sorry, Christmas laziness has come early!

Reply:
:christmas_tree: